src/rgw/rgw_op.cc

```
@@ -3914,7 +3914,7 @@ void RGWDeleteLC::execute()
   store->get_bucket_instance_obj(s->bucket, obj);
   store->set_atomic(s->obj_ctx, obj);
   ret = get_system_obj_attrs(store, s, obj, orig_attrs, NULL, &s->bucket_info.objv_tracker);
-  if (op_ret < 0)
```

it should be:
op_ret = get_system_obj_attrs
so we can return the error